jangjuseong/app.js

```diff
@@ -194,6 +199,31 @@ app.post('/likes/:userId/:postId', async (req, res) => {
   return res.status(201).json({ message: 'likeCreated' });
 });
 
+app.get('/users/signup', async (req, res) => {
+  const { email, password } = req.body;
```

Comment:
회원가입은 유저의 정보를 백엔드 서버로 보내기 때문에 post method로 사용해도 좋을 것 같습니다.

get을 사용하려면 request의 body로 데이터를 보내는 방식은 적절하지 않습니다.

메서드를 수정하는 것이 좋아보입니다. 추가로 signup 기능은 이미 있는 것 같은데 signin이 올바른 엔드포인트로 보입니다.

```suggestion
app.post('/users/signin', async (req, res) => {
  const { email, password } = req.body;
```